phenopy/config.py

```diff
@@ -80,6 +80,8 @@ def download(url, file_path):
 
 # create config directory if it doesn't exist
 config_directory = os.path.join(os.environ.get('HOME'), f'.{__project__}')
+project_directory = os.path.abspath(__project__)
+project_data_dir = os.path.join(project_directory, 'data')
```

Comment:
I don't think we need these we can just use `data_directory` defined in line 91.